common/utils.py

```diff
@@ -64,3 +64,28 @@ def file_hash(file_path):
             chunk = file_handle.read(chunk_size)
 
     return digest.hexdigest()
+
+
+def get_all_benchmarks():
```

Comment:
should be fuzzer_utils and benchmark_utils and not the super generic utils.py

Reply:
I think fuzzer utils is a bad module. Half the code is meant to run on the runner and half the code is meant to run on the dispatcher. get_all_fuzzers won't work on the runner even though the module fuzzer_utils is used. I need to break this up.

Reply:
How about moving to experiment_utils. i dont feel these function should get mixed with general utils stuff in this file like string and file hashes.

Reply:
I put them in fuzzer_utils and benchmark_utils for now.